node_modules/oae-core/editprofile/css/editprofile.css

```diff
@@ -0,0 +1,90 @@
+/*!
+ * Copyright 2015 Apereo Foundation (AF) Licensed under the
+ * Educational Community License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License. You may
+ * obtain a copy of the License at
+ *
+ *     http://opensource.org/licenses/ECL-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an "AS IS"
+ * BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+#editprofile-plea {
+    font-size: 14px;
+    font-style: normal;
+    margin-bottom: 15px;
+}
+
+#editprofile-email-verification .alert {
```

Comment:
> There's no need for this widget to use flexbox.

I'm sure you meant no offence in this choice of wording, but I find it very presumptuous and borderline insulting. A choice of words that gives the implementer a bit more credit that they went through a decision making process to choose an implementation is "Why was flexbox used as opposed to standard bootstrap and floats?", and I find this choice of words to be generally more common in your review feedback. Granted, I might be overly sensitive to my work ;)

That said, IIRC the "Resend email verification" button appears to have vertical centering in the design, and the content on the left is multi-line and likely wraps, so it is in essence variable height. Flexbox appeared to be the appropriate choice given the mechanics, and for IE9 only I thought it would be an acceptable trade-off to fix the button to the top as opposed to vertically center it. That's why I saw a need for flexbox, at least when I started implementing it.

If there is in fact a way to achieve vertical centering of the button while using bootstrap column classes, or if you'd prefer to ditch vertical centering entirely, let me know.